barretenberg/cpp/src/barretenberg/ultra_honk/decider_proving_key.cpp

```diff
@@ -316,12 +323,7 @@ void DeciderProvingKey_<Flavor>::move_structured_trace_overflow_to_overflow_bloc
 
     // Set the fixed size of the overflow block to its current size
     if (overflow_block.size() > overflow_block.get_fixed_size()) {
-        info("WARNING: Structured trace overflowed beyond the prescribed fixed overflow size. This is valid in the "
```

Comment:
this comment was only relevant for the initial compile time overflow mechanism. It doesn't apply since the introduction of dynamic overflow support.